pkgs/development/interpreters/python/hooks/pip-install-hook.sh

```diff
@@ -11,7 +11,9 @@ pipInstallPhase() {
     export PYTHONPATH="$out/@pythonSitePackages@:$PYTHONPATH"
 
     pushd dist || return 1
-    @pythonInterpreter@ -m pip install ./*.whl --no-index --prefix="$out" --no-cache $pipInstallFlags --build tmpbuild
+    mkdir tmpbuild
+    NIX_PIP_INSTALL_TMPDIR=tmpbuild @pythonInterpreter@ -m pip install ./*.whl --no-index --prefix="$out" --no-cache $pipInstallFlags
+    rm -rf tmpbuild
```

Comment:
Why delete the tmpbuild? Its still useful to have that in case a build fails and you want to inspect

Reply:
Oh right, because now its not a temp dir that is automatically clean up by pip...